.changelog/5566.bugfix.md

```diff
@@ -0,0 +1,4 @@
+governance: Remove unchanged proposal parameters
```

Comment:
```suggestion
governance/ChangeParametersProposal: Display only changed parameters

Omit the unchanged parameter values when pretty-printing the `ChangeParameterProposal`.
``` 

Maybe something like this, to make it clearer that this change only affects the pretty-printing of the proposals.